src/app.js

```diff
@@ -25,6 +25,10 @@ var OffsetToLineColumnConverter = require('./lib/offsetToLineColumnConverter')
 
 var examples = require('./app/example-contracts')
 
+// components
+var fileExplorer = require('_file-explorer')
```

Comment:
Oh, just a habit. I can make it a file.
I usually maintain a manually manage `src/node_modules` folder and prefix with underscore, so that i can require it without having to remember the path and whether i need to do `require('./modulename')` or `require('../modulename')`...  the underscore will clearly communicate that it is a project internal module, because stuff on `npm` can't contain `_` in their name.

It also makes it easy to later on publish internal modules as public ones on `npm` if they actually become something useful on their own :-)